src/styles/calendar.scss

```diff
@@ -0,0 +1,22 @@
+@import './index.scss';
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: $gap-size;
+  width: calc($day-size * 7 + $gap-size * 6); // 7 колонок
```

Comment:
Ensure that the variable `$day-size` is defined in the `index.scss` file or another imported SCSS file. Otherwise, this will cause an error during compilation.